src/device/usbd.c

```diff
@@ -196,24 +195,18 @@ static usbd_class_driver_t const _usbd_driver[] =
       .xfer_cb          = netd_xfer_cb,
       .sof              = NULL,
   },
-#endif
-  /* CDC-ECM management interface; CDC-EEM data interface */
+  /* CDC-ECM management interface */
   {
       .class_code       = TUSB_CLASS_CDC,
       .init             = netd_init,
       .reset            = netd_reset,
-#if CFG_TUD_NET == OPT_NET_EEM
-      .open             = netd_open_data,
-#else
       .open             = netd_open,
-#endif
       .control_request  = netd_control_request,
       .control_complete = netd_control_complete,
       .xfer_cb          = netd_xfer_cb,
       .sof              = NULL,
   },
   /* RNDIS/CDC-ECM data interface */
-#if CFG_TUD_NET != OPT_NET_EEM
   {
       .class_code       = TUSB_CLASS_CDC_DATA,
```

Comment:
if possible, please consider to merge them together as `cdc_device.c`, it is much cleaner and easier for class driver that way.